src/allmydata/test/test_connections.py

```diff
@@ -341,7 +341,7 @@ def setUp(self):
         self.config = config_from_string("fake.port", self.basedir, BASECONFIG)
 
     def test_default(self):
-        default_connection_handlers, _ = create_connection_handlers(None, self.config, mock.Mock(), mock.Mock())
+        default_connection_handlers, _ = create_connection_handlers(self.config, mock.Mock(), mock.Mock())
```

Comment:
I see that this call with mock instances is repeated over this test.
maybe refactor it into 

```suggestion
        default_connection_handlers = self.getConnectionHandlers(self.config)
```

where getConnectionHandlers will take care of create_connection_handlers, ignoring one return value and passing the stub arguments.